### DISCUSSION TITLE
how to unlink supabase account from github or delete supabase account?

### DISCUSSION THREAD

**Discussion by psyei4** [supabase/supabase#22757] [Questions]
Hi, so I've recently asked for support from supabase on deleting my account but it seems that users with free plans aren't put in priority? I've seen that the only way to delete the supabase account is through supabase but I'm wondering if there's any other way to unlink the account from github?

I will be waiting for the Supabase support response but just wanted to check as I have a coursework that requires the supabase account being separate to github so anything is appreciated.

**Comment by GaryAustin1** [comment]
I doubt unlinking works, if you need to create another account as it would be email based on your github email.

Can you use another email or it has to be from your github account?

Support is the only way I have seen.

**Reply by psyei4** [reply]
Yeah I think that I may have to try that if I don't get Support in time. 

I'll just wait for Support, just was looking to see if there was any other way. Thank you though!
